skiko/src/jvmMain/kotlin/org/jetbrains/skiko/GraphicsApi.kt

```diff
@@ -5,7 +5,7 @@ import org.jetbrains.skiko.redrawer.WindowsOpenGLRedrawer
 import org.jetbrains.skiko.redrawer.LinuxOpenGLRedrawer
 
 enum class GraphicsApi {
-    UNKNOWN, SOFTWARE, OPENGL, DIRECT3D, VULKAN, METAL
+    UNKNOWN, AWTSOFTWARE, SOFTWARE, OPENGL, DIRECT3D, VULKAN, METAL
```

Comment:
AWTSOFTWARE is not very descriptive, better keep SOFTWARE for AWTSOFTWARE and name the new software DIRECT_SOFTWARE. This way migration to new Skiko will not break things for people who relies on software in its current form.

Reply:
Done.